File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/centri_di_costo/ProdottoDaTrattareCDC.cs
namespace AgronicaCoreModelsSTD.attivita.centri_di_costo
{
    public class ProdottoDaTrattareCDC : CentroDiCosto
    {
        public MovimentoDiMagazzino giacenzaMagazzino {   get; set; }

        /// <summary>
        /// quantità prodotto interessata (può essere di meno rispetto a quella dello scarico) - Mov_Dettagli.QTA
        /// </summary>
        public decimal qtaTrattata { get; set; }

        public ProdottoDaTrattareCDC()
        {
            classType = costanti.ClassType.ProdottoDaTrattareCDC;
            tipo = Tipo.ProdottoDaTrattare;
        }
    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/centri_di_costo/ProdottoDaTrattareCDC.ts
import { CentroDiCosto, Tipo } from './CentroDiCosto';
import { MovimentoDiMagazzino } from 'app/Service/api.service';


export class ProdottoDaTrattareCDC extends CentroDiCosto {
    giacenzaMagazzino: MovimentoDiMagazzino;
    qtaTrattata: number;

    constructor() {
        super();
        this.classType = 'ProdottoDaTrattareCDC';
        this.tipo = Tipo.Esercizio;
    }
}
